TestVectorsAwsCryptographicMaterialProviders/runtimes/java/src/main/java/software/amazon/cryptography/materialproviders/internaldafny/wrapped/__default.java

```diff
@@ -27,6 +27,8 @@ > WrappedMaterialProviders(MaterialProvidersConfig config) {
       .builder()
       .impl(impl)
       .build();
-    return Result.create_Success(wrappedClient);
+    return software.amazon.cryptography.materialproviders.internaldafny._ExternBase___default.CreateSuccessOfClient(
+      wrappedClient
+    );
```

Comment:
HA! 

https://amzn-aws.slack.com/archives/C01PXG43J9E/p1711499109392679?thread_ts=1711490705.938219&cid=C01PXG43J9E

I thought that advice was bad!
And it was!
It is fine the client creation helpers to work in the abstract!

Reply:
I am happy to be proven wrong in this case. :)